Dynamo/Dynamo.SwiftLang/SLAttribute.cs

```diff
@@ -11,11 +11,22 @@ public SLAttribute (SLIdentifier name, CommaListElementCollection<SLBaseExpr> ar
 		{
 			Name = Exceptions.ThrowOnNull (name, nameof (name));
 			Add (new SimpleElememt ("@"));
+			StringRep = Name.Name;
```

Comment:
As a general rule of thumb, adding on to a string, especially in a loop is discouraged in .NET. It's a better practice to aggregate into a `StringBuilder` and do one `StringRep = builder.ToString();` at the end.

Reply:
Fixed